testing/citest/tests/google_server_group_test.py

```diff
@@ -161,6 +165,7 @@ def create_server_group(self):
       'instanceMetadata': {
         'load-balancer-names': self.__lb_name
       },
+      'userData': '='.join([self.__custom_user_data_key, self.__custom_user_data_value]),
```

Comment:
Though this produces the right output, I dont think it makes sense semantically because I'd read that as '=' over a list, but really it only makes sense on a 2-tuple (which it happens to be).  Instead I'd be more explicit with '{key}={value}'.format(key=..., value=...) (you dont necessarily need to name the parameters, {0} and {1} are fine)

FWIW, If it did need to be be more general ','.join(['{key}={value}'.format(key=key, value=value) for key,value in tupleList]) where tupleList is something like [('keyA', 'valueA'), ('keyB', 'valueB')]